src/agent/main.cpp

```diff
@@ -58,6 +54,12 @@
 #include "common/types.hpp"
 #include "ncp/thread_controller.hpp"
 
+#if OTBR_CONFIG_ANDROID_ENABLE
```

Comment:
Other OTBR macros are in the style `OTBR_ENABLE_*` though they're for features instead of platforms. Shall we make the name styles consistent?

Reply:
I named it to `OTBR_PLATFORM_ANDROID`